src/main/java/InOutApp.java

```diff
@@ -0,0 +1,41 @@
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class InOutApp {
+    static FileWriter fileWriter;
+    static BufferedWriter writer;
+
+    protected static double getResult(String operator, double number1, double number2) {
+        double result;
+        result = switch (operator) {
+            case "+" -> number1 + number2;
+            case "-" -> number1 - number2;
+            case "*" -> number1 * number2;
+            case "/" -> number1 / number2;
+            default -> 0;
+        };
+        return result;
+    }
+
+    protected static void writeResult(File file, double number1, String operator, double number2, double result) {
+        try {
+            fileWriter = new FileWriter(file, true);
+            writer = new BufferedWriter(fileWriter);
+            writer.write(number1 + " " + operator + " " + number2 + " = " + result);
+            writer.write(System.lineSeparator());
+            writer.flush();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
Co prawda zamykasz ten plik później, ale w przypadku błędu może pozostać niezamknięty. Najlepiej wprowadź try-with-resources https://youtu.be/LRwSHA9YDgY